core/systems/input/managed_gamepad.gd

```diff
@@ -368,6 +368,14 @@ func _process_phys_event(event: InputDeviceEvent, delta: float) -> void:
 				_send_input("ui_left", true)
 				axis_pressed = Bitwise.set_flag(axis_pressed, AXIS_PRESSED.LEFT)
 				return
+		event.ABS_RY:
+			var value := _normalize_axis(event)
+			_send_joy_input(JOY_AXIS_RIGHT_Y, value)
+			return
+		event.ABS_RX:
+			var value := _normalize_axis(event)
+			_send_joy_input(JOY_AXIS_RIGHT_X, value)
+			return
```

Comment:
ABS_Z and ABS_RZ are frequently used for L/R triggers, is that use case covered?